README.md

```diff
@@ -104,6 +104,7 @@ The following flags are available to toggle experimental features:
 
 - `newgraph` - From release v3.4.0. Improved graphing algorithm. (default `false`)
 - `meta` - From release v3.7.0. Show node metadata panel on click. (default `false`)
+- `lazy` - From release vx.x.x. Improved sidebar performance. (default `false`)
```

Comment:
It would be worth to actually put in `v.3.8.0` given that we have decided to release it in the next release to prevent this being forgotten to be updated in the release?